model/patch/patch.go

```diff
@@ -765,8 +765,9 @@ func (p *Patch) IsChild() bool {
 	return p.Triggers.ParentPatch != ""
 }
 
-// CollectiveStatus returns the aggregate status of all tasks and child patches.
-func (p *Patch) CollectiveStatus() (string, error) {
+// CollectivePatchStatus returns the aggregate status of all tasks and child patches.
+// If this is meant for display on the UI, we should also consider the display status aborted.
+func (p *Patch) CollectivePatchStatus() (string, error) {
```

Comment:
The previous name was actually more stylistically Go because it didn't [stutter](https://google.github.io/styleguide/go/best-practices#avoid-repetition). Can we change it back?

Reply:
hmm I just want to clarify that it returns the _patch_ status and not the version status since that's easy to forget those are different. If we make a follow up ticket to consolidate these, could we fix it at that point? Otherwise I can just leave a comment to be more aggressively specific about it.

Reply:
Well the tricky thing is I think we forget that PatchStatus and VersionStatus aren't the same, since that's what the bug was I found here.

Reply:
But yeah maybe I'll just make the comment aggressive and that could do the trick, plus we now know to be suspicious of these statuses.